modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -164,8 +152,22 @@ spec:
       v2:
         enabled: false
 ----  
-When using the full manifest, ensure that you choose the correct profile. In the example, the `default` profile is used.
+When using the full manifest, ensure that you choose the correct profile. In the example, the `default` profile is used. To install the manifest, execute:
++
+`istioctl manifest apply -f <manifest-file.yaml>`
++
+
+
+=== Install Istio 1.4.x
+
+To install and configure Istio 1.4.x for Anypoint Service Mesh:
 
+. Download Istio using the https://istio.io/docs/setup/[Istio Documentation].
```

Comment:
maybe?

```suggestion
. Download `istioctl` following the https://istio.io/docs/setup/[Istio Documentation].
```

Reply:
Not sure TBH, I think for customers that don't have previous Istio knowledge, it is better to download the folder and get the samples and tools.

Reply:
Sure, can we change only `using` -> `following`? seems better to me

Reply:
Leaving as is based on Tomas's confirmation.